go/libkb/secret_store_android.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// +build android
+
+package libkb
+
+import "strconv"
+
+func NewSecretStoreAll(mctx MetaContext) SecretStoreAll {
+	secFile := NewSecretStoreFile(mctx.G().Env.GetDataDir())
+	// Note: do not set up notifySecretStoreCreate for secret store file on
+	// Android, as it's only related to relevant to PGP key management.
+
+	if mctx.G().Env.ForceSecretStoreFile() {
+		// Allow use of file secret store on Android, for debugging or use with
+		// Termux (https://termux.com/).
+		return secFile
+	}
+
+	secAndroid := &secretStoreAndroid{}
+
+	var androidOsVersion int64
+	if v, err := strconv.ParseInt(mctx.G().MobileOsVersion, 10, 32); err != nil {
```

Comment:
may be better to do this in bind

Reply:
of course i messed this up, heh
should be
`if v, err := strconv.ParseInt(mctx.G().MobileOsVersion, 10, 32); err == nil`